.amplication/ignored/admin-ui/src/post/PostEdit.tsx

```diff
@@ -23,6 +23,9 @@ export const PostEdit = (props: EditProps): React.ReactElement => {
         </ReferenceInput>
         <TextInput label="Content" multiline source="content" />
         <TextInput label="Featured Image" source="featuredImage" />
+        <TextInput label="Meta Description" source="metaDescription" />
+        <TextInput label="Meta Title" source="metaTitle" />
+        <TextInput label="Slug" source="slug" />
         <ReferenceArrayInput
```

Comment:
You may want to add these manually to the actual files (note that this file in in the ignored folder)